spec/components/post_creator_spec.rb

```diff
@@ -1141,6 +1141,20 @@
       post_creator = PostCreator.new(user, title: '', raw: '')
       expect { post_creator.create! }.to raise_error(ActiveRecord::RecordNotSaved)
     end
+
+    it "does not generate " do
+      SiteSetting.queue_jobs = false
+      user2 = Fabricate(:user)
+      topic = Fabricate(:private_message_topic,
+        topic_allowed_users: [
+          Fabricate.build(:topic_allowed_user, user: user),
+          Fabricate.build(:topic_allowed_user, user: user2)
+        ],
+      )
+
+      PostAlerter.any_instance.expects(:after_save_post).never
```

Comment:
Is there another way to test this? We generally do not want to use stubs or mocks in our test suite.